charts/argus/Chart.yaml

```diff
@@ -6,7 +6,7 @@ maintainers:
   - email: argus@logicmonitor.com
     name: LogicMonitor
 name: argus
-version: 9.0.0-rc04
+version: 9.0.0-rc05
```

Comment:
argus chart 9.0.0 is already release, plz increment the chart version, say 9.1.0-rc01